webview/src/test/boundingBoxesFixture.ts

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable sonarjs/no-duplicate-string */
+import {
+  ComparisonBoundingBoxLabels,
+  ComparisonPlotBoundingBoxes,
+  ComparisonRevisionData,
+  PlotsComparisonData
+} from 'dvc/src/plots/webview/contract'
+
+const boxColors = [
+  '#ff3838',
+  '#ff9d97',
+  '#ff701f',
+  '#ffb21d',
+  '#cfd231',
+  '#48f90a',
+  '#92cc17',
+  '#3ddb86',
+  '#1a9334',
+  '#00d4bb',
+  '#2c99a8',
+  '#00c2ff',
+  '#344593',
+  '#6473ff',
+  '#0018ec',
+  '#8438ff',
+  '#520085',
+  '#cb38ff',
+  '#ff95c8',
+  '#ff37c7'
+]
+
+const boundingBoxImgLabels: ComparisonBoundingBoxLabels = {
+  car: { color: boxColors[1], selected: true },
+  sign: { color: boxColors[2], selected: false },
+  'traffic light': { color: boxColors[3], selected: true }
+}
+
+const boundingBoxImgCoords: { [rev: string]: ComparisonPlotBoundingBoxes } = {
+  'exp-83425': [
+    { boxes: [{ h: 75, w: 100, x: 100, y: 100 }], label: 'traffic light' },
+    { boxes: [{ h: 30, w: 30, x: 190, y: 310 }], label: 'car' }
+  ],
+  'exp-e7a67': [
+    { boxes: [{ h: 110, w: 100, x: 90, y: 100 }], label: 'traffic light' },
+    { boxes: [{ h: 30, w: 30, x: 190, y: 310 }], label: 'car' }
+  ],
+  main: [
+    { boxes: [{ h: 100, w: 100, x: 100, y: 100 }], label: 'traffic light' },
+    { boxes: [{ h: 30, w: 30, x: 190, y: 310 }], label: 'car' }
+  ],
+  'test-branch': [
+    { boxes: [{ h: 100, w: 90, x: 100, y: 110 }], label: 'traffic light' },
+    { boxes: [{ h: 30, w: 30, x: 190, y: 310 }], label: 'car' }
+  ],
+  workspace: [
+    { boxes: [{ h: 90, w: 80, x: 120, y: 120 }], label: 'traffic light' },
+    {
+      boxes: [
+        { h: 30, w: 30, x: 190, y: 310 },
+        { h: 50, w: 60, x: 300, y: 320 }
+      ],
+      label: 'car'
+    }
+  ]
+}
+
+export const addBoundingBoxes = (
```

Comment:
I've created a util in the frontend that adds the bounding boxes to a comparison fixture for testing and storybooks. Once we've got the backend part of this working (https://github.com/iterative/dvc/issues/10198), this will all be removed and bounding box data will be part of our original comparison fixture.